kratos/includes/registry.h

```diff
@@ -104,6 +104,13 @@ class KRATOS_API(KRATOS_CORE) Registry final
         KRATOS_CATCH("")
     }
 
+    template<typename... Types>
+    static std::string RegistryTemplateToString(Types&&... args) {
+        std::string f_name = (... += ("," + std::to_string(args)));
+        f_name.erase(0,1);
+        return f_name;
```

Comment:
How about this:

```suggestion
        return (std::to_string(args) + ...);
```

https://godbolt.org/z/8bs1n5Gex

Reply:
ah you need the comma as delimiter, nevermind

Reply:
yep sadly... I am not the best out there dealing with strings in C++ :P. It also hurts my eyes, I will change it as soon as I find something better